coverage.out

```diff
@@ -0,0 +1,75 @@
+github.com/maxence-charriere/go-objcruntime/class.go:42.58,47.2 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:49.91,57.2 5 0
+github.com/maxence-charriere/go-objcruntime/class.go:59.54,65.47 4 0
+github.com/maxence-charriere/go-objcruntime/class.go:75.2,75.8 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:65.47,69.39 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:69.39,72.4 2 0
+github.com/maxence-charriere/go-objcruntime/class.go:78.45,80.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:82.53,84.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:86.49,88.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:90.57,92.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:94.57,99.2 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:101.64,107.47 4 0
+github.com/maxence-charriere/go-objcruntime/class.go:117.2,117.8 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:107.47,111.39 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:111.39,114.4 2 0
+github.com/maxence-charriere/go-objcruntime/class.go:120.71,125.2 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:127.66,129.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:131.63,133.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:135.57,141.47 4 0
+github.com/maxence-charriere/go-objcruntime/class.go:151.2,151.8 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:141.47,145.39 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:145.39,148.4 2 0
+github.com/maxence-charriere/go-objcruntime/class.go:154.74,159.2 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:161.61,163.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:165.67,167.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:169.56,171.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:173.59,175.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:177.85,191.37 9 0
+github.com/maxence-charriere/go-objcruntime/class.go:202.2,202.91 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:191.37,200.3 6 0
+github.com/maxence-charriere/go-objcruntime/class.go:205.84,219.37 9 0
+github.com/maxence-charriere/go-objcruntime/class.go:230.2,230.83 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:219.37,228.3 6 0
+github.com/maxence-charriere/go-objcruntime/class.go:233.66,235.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:237.63,243.47 4 0
+github.com/maxence-charriere/go-objcruntime/class.go:253.2,253.8 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:243.47,247.39 3 0
+github.com/maxence-charriere/go-objcruntime/class.go:247.39,250.4 2 0
+github.com/maxence-charriere/go-objcruntime/class.go:256.43,258.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:260.52,262.2 1 0
+github.com/maxence-charriere/go-objcruntime/class.go:265.47,268.2 2 0
+github.com/maxence-charriere/go-objcruntime/class.go:270.53,273.2 2 0
```

Comment:
I think you shouldn't commit this file, add it to the `.gitignore` so it doesnt' get added automatically.